nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -1131,7 +1131,11 @@ define([
             }
             const state = fsm.getCurrentState();
             try {
-                FSMBar.showFsmBar({ ui: ui, state: state, job: model.getItem('exec.jobState') });
+                FSMBar.showFsmBar({
```

Comment:
I guess Prettier and a stray git commit snuck something in here.

Reply:
I sometimes wonder if Prettier changes its mind between runs.